### ISSUE TITLE
Update stars on tap

### ISSUE THREAD

**Issue by dcortright** [evgenyneu/Cosmos#96]
Is there a way to update the stars using just a tap? It seems as though you either have to scroll across or hold down for longer than a tap.

If there is a way to do this I would love to know! Or if there isn't, could it be a possible future feature?

* Library setup method: CocoaPods
* Version of the library: 14.0.0
* Xcode version. Example: 9.2
* OS version. Example: iOS 11

**Comment by evgenyneu** [comment]
Thanks for reporting, @dcortright. It looks like a bug because Cosmos is supposed to update the rating on just a quick tap as well. How are you using cosmos? There is a demo app in this repo, it would be helpful if you could check if the problem exists there for you as well. Let me know how it goes.

**Comment by dcortright** [comment]
I have a view that I am presenting on a custom alert that provides the user the rating stars as well as a text box for entering a review. I tested a different view that also uses Cosmos and I am able to tap and update the stars there. So it is something with presenting it on the alert. I will report back once I determine the bug. Thank you for responding!

**Comment by evgenyneu** [comment]
Interesting, something may be blocking the tap gesture.

> I will report back once I determine the bug.

It will be great, thanks. :)

**Comment by dcortright** [comment]
It the alert controller that I am using that is blocking the tap gesture. Off to find out which one will have to change. Thanks again!

**Comment by evgenyneu** [comment]
No problem, feel free to share your solution here if you find out.

**Comment by dcortright** [comment]
Alright. In case anyone else runs into this problem this is what I had to do:

I was using a combination of CFAlertViewController and Cosmos to present a view to the user that allows them to create a review on a product. 

BUT...CFAlert listens for taps on its view so that if you tap outside the view it will dismiss it. So what I did was remove all of the gesture recognizers from the CFAlert view and instead will handle them on my own.

```
for recognizer in alertController.view.gestureRecognizers ?? [] {
            alertController.view.removeGestureRecognizer(recognizer)
        }
```

The stars are back to acting as they should! Thank you @evgenyneu
